src/mds/MDCache.cc

```diff
@@ -204,8 +206,17 @@ void MDCache::handle_conf_change(const ConfigProxy& conf,
 {
   if (changed.count("mds_cache_size"))
     cache_inode_limit = g_conf().get_val<int64_t>("mds_cache_size");
+  if (changed.count("mds_memory_target")){
+    if(cache_autotune) {
+      cache_memory_limit = g_conf().get_val<Option::size_t>("mds_memory_target")/1.25;
```

Comment:
We don't need to change this here. `MDCache::resize` should be the only place we manipulate `cache_memory_limit`. (The default value is still setup by `MDCache::get_default_mds_cache_memory_limit()`.)

So I think we can drop this entire block.